<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderToFileDialog</class>
 <widget class="QDialog" name="RenderToFileDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>308</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Render to files</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QLineEdit" name="xResolution"/>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="xLabel">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLineEdit" name="yResolution"/>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="pixelsLabel">
     <property name="text">
      <string>pixels</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="8">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="10" column="5" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="fileLabel">
     <property name="text">
      <string>File name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="folderLabel">
     <property name="text">
      <string>Destination folder</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="folderEdit"/>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="msgLabel">
     <property name="text">
      <string>Select the destination folder and the rendering options:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="fileEdit">
     <property name="toolTip">
      <string>%0d: frame number with leading zeros; %%: percent; numeric printf formats accepted</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="firstFrameLabel">
     <property name="text">
      <string>First frame</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="firstFrameEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="resolutionLabel">
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="timeOffsetLabel">
     <property name="text">
      <string>with time offset</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="startPageLabel">
     <property name="text">
      <string>Start page</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="startPageEdit">
     <property name="toolTip">
      <string>0 for current page</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLineEdit" name="timeOffsetEdit">
     <property name="toolTip">
      <string>The value of page_time for the first frame in start page</string>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="QLabel" name="secondsLabel2">
     <property name="text">
      <string>seconds</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="durationLabel">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="durationEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QLabel" name="secondsLabel">
     <property name="text">
      <string>seconds</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="startTimeLabel">
     <property name="text">
      <string>Start time</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="startTimeEdit">
     <property name="toolTip">
      <string>Unix time in seconds, -1 for current date/time</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="fpsLabel">
     <property name="text">
      <string>Frames per second</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="fpsEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>folderEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>fileEdit</tabstop>
  <tabstop>firstFrameEdit</tabstop>
  <tabstop>xResolution</tabstop>
  <tabstop>yResolution</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RenderToFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RenderToFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
